apps/examples/testcase/le_tc/compression/tc_compress_read.c

```diff
@@ -22,12 +22,51 @@
 #include <tinyara/config.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <errno.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <tc_common.h>
 #include <tinyara/os_api_test_drv.h>
+#include <tinyara/compression.h>
 #include "tc_internal.h"
 
+#define MAX_INPUT_SIZE 10000
+#define MAX_OUTPUT_SIZE 10000
```

Comment:
If we set different value with input size, is it working? If not, how about `#define MAX_OUPTU_SIZE MAX_INPUT_SIZE`?

Reply:
it should be of same size according to compression algorithm , so #define MAX_OUTPUT_SIZE MAX_INPUT_SIZE done